### PR TITLE
A few development quality-of-life improvements

### PR DESCRIPTION
**Pull request by spladug**
These address #22 and #24 and as a bonus make it a little harder to end up with typos in the docs.

### CODE REVIEW

**Comment by spladug** [comment]
:eyeglasses: @rram @ckwang8128

File: docs/words.txt [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+gevent
 Gevent
+healthcheck
 hostname
 INI
 lifecycle
 namespace
+predecided
 pubsub
+Queing
```
Comment (ckwang8128) [inline docs/words.txt]: unless this is a proper noun, I believe the spelling is "queuing". 

question for learning: what's this file do?

**Comment by ckwang8128** [comment]
:nail_care: 

Looks good!

File: docs/words.txt (hunk 1 above)
Comment (spladug) [inline docs/words.txt]: http://english.stackexchange.com/questions/60852/queueing-or-queuing 

I can switch to the long string of vowels form if you think it'd be more standard though.

This file is a custom dictionary for the sphinx spellchecker: http://sphinxcontrib-spelling.readthedocs.org/en/latest/

**Comment by spladug** [comment]
:haircut:

**Comment by ckwang8128** [comment]
:fish: english is fun

**Comment by spladug** [comment]
c6e0065f0d643dc2aaeeaad1925f88fb57c6d4a5...a72f3ce44bdba0630c71cd324c827f1ed24ddfd7

**Pull request closed without merging**